main.py

```diff
@@ -265,6 +265,9 @@ def ui_access_wrapper(self, *args, **kwargs):
     get_redirect_route(
         r'/usercommunityrightsdatahandler',
         community_dashboard.UserCommunityRightsDataHandler),
+    get_redirect_route(
+        r'/getfeaturedtranslationlanguages',
```

Comment:
I've left a comment on top for the name of this url, we can change it to the name of the controller  `featuredtranslationlanguageshandler`, what do you think?

Reply:
Done